### DISCUSSION TITLE
I want to embed the dashboard in my application .How can i do it

### DISCUSSION THREAD

**Discussion by mkrajirajan** [getredash/redash#6722] [Q&A] [answered]
I want to embed the dashboard in my application .How can i do it

**Comment by gaecoli** [comment, marked as answer]
> I want to embed the dashboard in my application .How can i do it

At dashboard page to click 
<img width="95" alt="image" src="https://github.com/getredash/redash/assets/46274612/08b6b999-b8d6-4961-8cec-2caa8347ead0">
